fixtures/backup/fresh-install.json

```diff
@@ -134,9 +134,7 @@
   "model": "sentry.useremail",
   "pk": 1,
   "fields": {
-    "user": [
-      "testing@example.com"
-    ],
+    "user": 1,
```

Comment:
Is this related to https://github.com/getsentry/sentry/pull/54477/files#r1290752638?

Reply:
It is! Before, when we `use_natural_keys=True` on export, we'd identify users by the their natural key when serializing them to JSON, as seen above. `User` was the only model that did this, because it inherits from a native Django model, `AbstractBaseUser`, that does it. None of Sentry's own models do this.